content/_releases/22/20.md

```diff
@@ -0,0 +1,165 @@
+description: Details of the product update v22.20 from 19th of May 2022.
+layout: rel
+version: "22.20"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-05-19
+redirect_from:
+  - /releases/22.20.html
+---
+
+## Improvements and Updates
+
+### Re-authentication for sensitive actions
+
+While working on the platform user interface we try to click our way through
+setup options and there is a chance we might accidentally remove a sensitive data.
+There can be also cases when your account might be compromised and somebody would
+deliberately try to cause harm.
+
+For such cases we introduce re-authentication for sensitive actions. After
+{{page.version}} for the following actions you will be asked to enter your password
+to authenticate:
+
+*   Reading, copying and regenerating your API Keys.
+*   Modifying your user details.
+*   Disabling 2-Factor Authentication (2FA).
+*   Inviting members to contract.
+*   Inviting and removing members from workspaces.
+
+{% include img.html max-width="40%" url="/assets/img/RN/2220/RN2220-reauth.png" title="Sensitive action re-authentication" %}
+
+> **Please Note**: You will be asked to re-authenticate once every 6 hours.
+
+### Email verification during registration
+
+To prevent erroneous and bot registrations we introduce email verification step to
+our registration process. When you register first time with our platform, an email
+will be sent to your given email address. Click on the link in your received email
+to finish your registration. There are certain rules and restrictions to this process as well:
+
+*   If you register using Single-Sign-on (SSO) Provider (OIDC/SAML) then platform will not ask to verify your email address since you have verified it with your provider.
+*   You must wait for 30 minutes before you can use the same email address to register.
+*   The invitation to confirm your email is valid only for 1 day.
+
+> **Please Note**: If you are invited to the platform directly, no email verification will be required.
+
+{% if site.data.tenant.name == "elastic.io" %}
+
+## HELM3
+
+We constantly improve different aspects of HELM3 deployments in collaboration
+with our partners. This section lists updates and improvements done in this round.
+
+### Separate MongoDB for Maester
+
+When your integration flow needs to transfer big objects or attachments, one of
+the platform microservices called Maester temporary stores them in MongoDB while
+passing only the object IDs through the integration flow steps.
+
+Starting from {{page.version}}, you can use a dedicated database for storing Maester
+objects and the run-time attachments. To configure, add `MAESTER_MONGO_URI`
+environment variable to HELM3 secrets charts and set it to target this new database.
+
+> Note: You will need to migrate the grids to new MongoDB as well to ensure the Maester
+> object availability after implementation.
+
+If you don't need to use a separate MongoDB to store Maester object set
```

Comment:
Organisations who do not need a separate MongoDB instance to store their Maester objects should set the 'MAESTER_MONGO_URI' variable to the same value as the 'MONGO_URI' variable.

Reply:
Corrected as requested.